### PR TITLE
feat(Theming): add `VisibilityByTheme` as a shared component

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

**Comment by snorrekim** [comment]
Insted of `FilterByThem.Exclude`, we could have the props `include` and `exclude` instead of  `name`

**Comment by snorrekim** [comment]
Or perhaps `visible` and `hidden` would be better names, then there's no confusion about what "Filter" means, are we filtering out or in?

`<FilterByTheme name="sbanken">` doesn't really say if sbanken is filtered out or in.

**Comment by tujoworker** [comment]
I think your suggestion is very good! For better TypeScript support, we should use arrays. So moved this component to be a part of Eufemia, added tests and docs 👍

File: packages/dnb-design-system-portal/src/docs/contribute/style-guides/documentation.mdx [hunk 1]
```diff
@@ -13,13 +13,13 @@ The documentation is written in enhanced Markdown, called MDX. It allows us to i
 If you need to show some documentation only for when a certain theme is choose, you can do so:
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/contribute/style-guides/documentation.mdx:13]:
```suggestion
If you need to show some documentation only for when a certain theme is chosen, you can do so:
```

Comment (langz) [inline packages/dnb-design-system-portal/src/docs/contribute/style-guides/documentation.mdx:13]:
```suggestion
If you need to show some documentation only for when a certain theme is selected, you can do so:
```

File: packages/dnb-design-system-portal/src/docs/uilib/components/button/info.mdx [hunk 2]
```diff
@@ -33,10 +33,14 @@ It is **not** recommended to use the tertiary button without an icon. Looking fo
 
 <TertiaryButtonSizes />
 
+<VisibilityByTheme hidden="sbanken">
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/button/info.mdx:36]: Nice nice, we should remember to add this to the demos for variant signal as well 🙏 
Maybe in a upcoming PR

**Review by langz — APPROVED** [review]

File: packages/dnb-design-system-portal/src/docs/uilib/usage/customisation/theming.mdx [hunk 3]
```diff
@@ -46,11 +46,48 @@ render(
...
+  </Theme>
+)
+```
+
 `<Theme>` Will create a `div` wrapper by default, when no custom element is defined (`element="span"`). It sets these CSS classes:
 
 - `eufemia-theme__{theme-name}`
 - `eufemia-theme__{theme-name}--{theme-variant}`
 
+###
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/usage/customisation/theming.mdx:89]:
```suggestion
```

**Review by langz — APPROVED** [review]

**Review by joakbjerk — APPROVED** [review]
Looks good to me 👍

**Pull request merged**